Documentation/QtCreatorConfiguration.md

```diff
@@ -78,7 +78,7 @@ In order to so, create a new file anywhere, for example `license-template.creato
 
 ```
 /*
- * Copyright (c) 2023, the SerenityOS developers.
```

Comment:
updating the year here is fine, but changing from SerenityOS developers to Ladybird developers feels odd. we don't have that line in the Liscense.md. not sure what to do about it tbh. we can't just remove them from it either :)